sdks/java/core/src/test/java/org/apache/beam/sdk/util/HistogramDataTest.java

```diff
@@ -200,4 +202,134 @@ public void testIncrementBucketCountByIndex() {
     assertThat(data.getTopBucketCount(), equalTo(4L));
     assertThat(data.getTotalCount(), equalTo(10L));
   }
+
+  // The following tests cover exponential buckets.
+  @Test
+  public void testPositiveScaleBucket() {
```

Comment:
maybe rename the tests that test exponential buckets to

`testExponentialBuckets_{TEST_CASE}`
for example `testExponentialBuckets_positiveScaleBuckets()`

Reply:
Done.